setup.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: page
+title: Flutter Setup
+nav_title: Flutter Setup
+permalink: /setup/
+---
+
+This guide describes how to set up your development environment to run Flutter apps on
+iOS or Android.
+
+[System requirements](#system_requirements)<br>
+[iOS setup](#ios_setup)<br>
+[Android setup](#android_setup)<br>
+[Get the Flutter SDK](#get_the_flutter_sdk)<br>
+[Atom editor](#atom_editor)
+
+## System requirements ##
```

Comment:
you use `## foo ##` here but other files (including the other one in this patch) only use `## foo`.